test/system/beta/clipboard_copy_button_test.rb

```diff
@@ -19,7 +19,7 @@ def test_copies_text
     def test_includes_tooltip
       visit_preview(:with_tooltip)
 
-      assert_selector ".Button + tool-tip", visible: :all
+      assert_selector "tool-tip[for='clipboard-button']", visible: :all
```

Comment:
Can we add a system test that ensures the aria-live region gets populated with the "Copied!" text when the button is clicked?